ee/clickhouse/models/cohort.py

```diff
@@ -57,7 +57,10 @@ def format_person_query(
         from ee.clickhouse.queries.cohort_query import CohortQuery
 
         query, params = CohortQuery(
-            Filter(data={"properties": cohort.properties}), cohort.team, cohort_pk=cohort.pk, cohorts_seen=cohorts_seen
+            Filter(data={"properties": cohort.properties}, team=cohort.team),
```

Comment:
without the team, we weren't simplifying filters at all! :$

This will probably change a few snapshots atleast, will fix once I find out which ones 👀